notebooks/official/pipelines/google_cloud_pipeline_components_model_upload_predict_evaluate.ipynb

```diff
@@ -0,0 +1,986 @@
+{
```

Comment:
<p>wrong json file to remove -&gt; valuation_demo_pipeline.json</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/288/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='53'/>